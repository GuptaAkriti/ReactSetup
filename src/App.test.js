import React from 'react';
import App from './App';

import { shallow } from 'enzyme';

describe('App Component', () => {
    it('renders the Counter wrapper', () => {
      const wrapper = shallow(<App />);
      expect(wrapper.getElements()).toMatchSnapshot();
    });
})