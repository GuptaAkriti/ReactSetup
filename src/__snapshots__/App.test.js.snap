// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders the Counter wrapper 1`] = `
Array [
  <p>
    hello
  </p>,
]
`;
